exercises/concept/tim-from-marketing-2/src/test/java/EmployeeServiceTest.java

```diff
@@ -0,0 +1,75 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Optional;
+
+// Annotation in order to use @BeforeAll in a non-static method
+@TestInstance(Lifecycle.PER_CLASS)
+public class EmployeeServiceTest {
+
+    private EmployeeService employeeService;
+    private List<Optional<Employee>> listOfEmployees = new ArrayList<>();
+
+    private List<String> expectedListOfPrintedEmployeesNames = List.of(
+            "0 - Tim\n" + "1 - No employee found\n" + "2 - John\n"
+    );
+
+    private List<String> expectedListOfPrintedEmployeesNamesAndDepartments = List.of(
+            "0 - Tim - Direction",
+            "1 - No employee found",
+            "2 - John - Engineering"
+    );
+
+    void initList() {
+        listOfEmployees.add(Optional.of(new Employee(0, "Tim", "Direction")));
+        listOfEmployees.add(Optional.empty()); // Adding empty value.
+        listOfEmployees.add(Optional.of(new Employee(2, "John", "Engineering")));
+    }
+
+    @BeforeAll
```

Comment:
As mentioned in another comment, changing this should get the tests passing in the CI.

```suggestion
    @BeforeEach
```